openadapt/events.py

```diff
@@ -523,29 +564,31 @@ def get_merged_events(to_merge, state):
 
         return merged_events
 
-
     return merge_consecutive_action_events(
-        "redundant_mouse_move", events, is_target_event, get_merged_events,
+        "redundant_mouse_move",
+        events,
+        is_target_event,
+        get_merged_events,
     )
 
 
 def merge_consecutive_action_events(
-    name, events, is_target_event, get_merged_events,
-):
-    """Merge consecutive action events into a single event"""
-
+    name: Any,
+    events: Any,
+    is_target_event: Any,
+    get_merged_events: Any,
+) -> list:
+    """Merge consecutive action events into a single event."""
     num_events_before = len(events)
     state = {"dt": 0}
     rval = []
     to_merge = []
 
-
-    def include_merged_events(to_merge):
+    def include_merged_events(to_merge: Any) -> None:
```

Comment:
`to_merge: list[ActionEvent]`

Reply:
Addressed in https://github.com/OpenAdaptAI/OpenAdapt/pull/289/commits/d88767ff785961d7eb36ca589d96f707a019b507